Sources/TuistCloud/OpenAPI/Client.swift

```diff
@@ -0,0 +1,80 @@
+// Generated by swift-openapi-generator, do not modify.
+@_spi(Generated) import OpenAPIRuntime
+#if os(Linux)
+@preconcurrency import Foundation
+#else
+import Foundation
+#endif
+public struct Client: APIProtocol {
+    /// The underlying HTTP client.
+    private let client: UniversalClient
+    /// Creates a new client.
+    /// - Parameters:
+    ///   - serverURL: The server URL that the client connects to. Any server
+    ///   URLs defined in the OpenAPI document are available as static methods
+    ///   on the ``Servers`` type.
+    ///   - configuration: A set of configuration values for the client.
+    ///   - transport: A transport that performs HTTP operations.
+    ///   - middlewares: A list of middlewares to call before the transport.
+    public init(
+        serverURL: URL,
+        configuration: Configuration = .init(),
+        transport: any ClientTransport,
+        middlewares: [any ClientMiddleware] = []
+    ) {
+        self.client = .init(
+            serverURL: serverURL,
+            configuration: configuration,
+            transport: transport,
+            middlewares: middlewares
+        )
+    }
+    private var converter: Converter { client.converter }
```

Comment:
What's a converter?

Reply:
I ... don't know 😄 this is the generated file that we don't directly control.

Reply:
AFAICT, converter takes in the raw JSON and converts it to Swift models as defined by the OpenAPI specification.